styles/_mixins.scss

```diff
@@ -22,3 +22,25 @@
   pointer-events: none;
   opacity: 0.2;
 }
+
+// todo: refactor this style once we have designs
+@mixin processingOverlay {
+  position: relative;
```

Comment:
yeah, just spent an hour on this and I just can't get that darn overlay to extend. If you want to play around and see if you can come up with a way, let me know.

Otherwise, I think another option is to disable the scroll bar. Since in css there's no way to disable a scroll bar without hiding it (which would mean a page jump), I'd need to use some JS to get the scrollbar width and apply a margin to compensate for the hidden scroll bar. But... don't think that would be too difficult.